✨ Adding support for windows terminal

themegabyte opened the pull request:
Hello there,

I saw there is no support for [Windows Terminal](https://github.com/microsoft/terminal). Hence I traced back my own steps and I am submitting a pull request!

Here are some things to consider,

1. magenta has been mapped to "brightPurple" because WT doesn't have magenta.
2. the links in both readme are "relative" instead of absolute ones like before.
3. Since there wasn't any color for "selectionBackground" I put it to: [`#9cb0b9`](https://www.color-hex.com/color/9cb0b9)

Thank you!

themegabyte committed:
Replaced magenta with bright purple.

bumped "selectionBackground" slightly up because it was not visible.
text color has been assigned to "foreground"

themegabyte committed:
Adding link to instructions for windows termnial.

themegabyte committed:
Adding instructions for Windows Terminal

themegabyte committed:
Added Material Black.

themegabyte committed:
Noticed there is a "background" entry as well. Mapping it correctly.

themegabyte committed:
Fixed bright black to be the same as "text" so it isn't too dark.

Windows powershell uses brightblack as command line switches for commands.

themegabyte committed:
Fixing typo in the heading and the hyperlink

The link on the main Readme.md will navigate to the windows terminal heading correctly.

carloscuesta commented on shell-color-themes/windows/windowsterminal/dark.json:
```diff
@@ -0,0 +1,23 @@
+{
```
Hey! Since the folder is already named `windows` I think we should rename `windowsterminal` to `terminal` ?

Wdyt 🤔

themegabyte replied:
I think that's a good catch, thank you!

carloscuesta reviewed (commented):
Thanks for submitting a PR! 👀

themegabyte committed:
fixing redundancy in folder name.

renaming windowsterminal to terminal because the folder one level up is also named "windows".

path windows/terminal is better then windows/windowsterminal.

carloscuesta reviewed (approved):
LGTM! 🚀
